packages/@aws-cdk/core/lib/custom-resource-provider-handler.ts

```diff
@@ -0,0 +1,139 @@
+// tslint:disable: no-console
```

Comment:
Not sure, but I don't think there should be a whitespace after `disable:`. can you check?

Also there's only one line here using it. Would be preferrable to move it to above the line and use `// tslint:disable-next-line`

Reply:
- The whitespace is ok (and optional).
- This is intentionally for the entire file. The handler is *expected* to use `console.log` in order to emit logs to cloudwatch.

Reply:
okay, now I see that all logs are funneled through a single `log` function, so you are right. Also removed the extraneous whitespace because it's ugly.